### ISSUE TITLE
Needs to cancel task

### ISSUE THREAD

**Issue by pablomelo** [nuveo/gofn#42]
As a user,
I want to be able to cancel a running task,
Because sometimes I make mistakes.

**Comment by avelino** [comment]
I don't think it would be the responsibility of the gofn manage processes, we can create the gofn-server
